services/common/src/components/syncfusion/DocumentViewer.tsx

```diff
@@ -85,22 +92,46 @@ interface ViewPDFProps {
   pdfViewerServiceUrl: string;
   documentPath: string;
   ajaxRequestSettings: any;
+  id?: string;
+  onInit?: (pdfViewer: any) => void;
+}
+interface PDFViewerProps {
+  documentPath: string;
+  id?: string;
+  onInit?: (pdfViewer: any) => void;
 }
 
-const ViewPdf: React.FC<ViewPDFProps> = ({
+export const PdfViewer: React.FC<PDFViewerProps> = (props: PDFViewerProps) => {
+  const ajaxSettings = getAjaxRequestSettings();
+
+  return <ViewPdf id={props.id} onInit={props.onInit} pdfViewerServiceUrl={ENVIRONMENT.pdfViewerServiceUrl} documentPath={props.documentPath} ajaxRequestSettings={ajaxSettings} />;
+};
+
+export const ViewPdf: React.FC<ViewPDFProps> = ({
   pdfViewerServiceUrl,
   documentPath,
   ajaxRequestSettings,
+  id = "pdfviewer-container",
+  onInit = null
 }) => {
+  let pdfViewer;
   return (
     <PdfViewerComponent
-      id="pdfviewer-container"
+      id={id || "pdfviewer-container"}
```

Comment:
Could probably remove the "pdfviewer-container" part here since it's set as a default in the props.